rosplan_planner_interfaces/launch/ffha_planner_interface.launch

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<launch>
+
+	<!-- arguments -->
+	<arg name="node_name"            default="rosplan_planner_interface" />
+	<arg name="use_problem_topic"    default="true" />
+	<arg name="problem_topic"        default="/rosplan_problem_interface/problem_instance" />
+	<arg name="planner_topic"        default="planner_output" />
+	<arg name="domain_path"          default="$(find rosplan_planning_system)/test/pddl/turtlebot/domain.pddl" />
```

Comment:
this domain has durative actions, which are not supported neither by ff nor ffha. I suggest we change it e.g. for ROSPlan/rosplan_planning_system/test/pddl/amazon